cves/CVE-2016-2161.yml

```diff
@@ -152,13 +154,12 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer:
+  answer: Other than the first commit listed above in the interesting_commits section, the changes made to this file were quite small and only consisted of small updates.
```

Comment:
What major event is this answer talking about? I'm confused what `this file` refers to